Badge.md

```diff
@@ -1,3 +1,139 @@
 (Version >= 1.16.0)
+With https://github.com/louislam/uptime-kuma/pull/1119
```

Comment:
Other sections of the wiki also don't have the PR they are added in

```suggestion
```

Reply:
ok, so we delete here